sql: deflake the logic test `event_log`

knz opened the pull request:
Fixes #35463

DDL operations can retry, if they do the table/db IDs will be
non-deterministic. The test can't assert specific IDs for this reason.

Release note: None

knz committed:
sql: deflake the logic test `event_log`

DDL operations can retry, if they do the table/db IDs will be
non-deterministic. The test can't assert specific IDs for this reason.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/35534)
<!-- Reviewable:end -->

knz commented:
Feel free to merge on green and/or take ownership to fix any remaining CI failures (I'm flying and probably OOO tomorrow).

BramGruneir commented:
Lgtm, thanks for the super quick fix!

On Thu, Mar 7, 2019, 20:35 kena <notifications@github.com> wrote:

> Feel free to merge on green and/or take ownership to fix any remaining CI
> failures (I'm flying and probably OOO tomorrow).
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/35534#issuecomment-470769828>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABihueg1pY8Y2Mvd9MhHaCJgyoSd-6gNks5vUb5GgaJpZM4bkdmS>
> .
>

knz committed:
sql: ensure that auto-gen idx names go to event_log

Release note (bug fix): CockroachDB now properly preserves the
automatically generated name of a newly created index in
`system.event_log` when the name is not specified in the `CREATE INDEX`
statement.

knz commented:
TFYR!

bors r+

knz commented:
bors r+

knz commented:
bors r+
